.gitignore

```diff
@@ -16,3 +16,6 @@ target/
 
 # results from HTTP API testing tool
 *run-*
+
+# Intelij Idea files
+.idea/*
```

Comment:
Hey @lunaryorn , 
Thanks for the suggestion, I was not aware of the global ignore file!